charts/keess/templates/deployment.yaml

```diff
@@ -0,0 +1,77 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ include "keess.fullname" . }}
+  namespace: {{ .Release.Namespace }}
+  labels:
+    {{- include "keess.labels" . | nindent 4 }}
+spec:
+  selector:
+    matchLabels:
+      {{- include "keess.selectorLabels" . | nindent 6 }}
+  template:
+    metadata:
+      {{- with .Values.podAnnotations }}
+      annotations:
+        {{- toYaml . | nindent 8 }}
+      {{- end }}
+      labels:
+        {{- include "keess.selectorLabels" . | nindent 8 }}
+    spec:
+      {{- with .Values.imagePullSecrets }}
+      imagePullSecrets:
+        {{- toYaml . | nindent 8 }}
+      {{- end }}
+      serviceAccountName: {{ include "keess.serviceAccountName" . }}
+      securityContext:
+        {{- toYaml .Values.podSecurityContext | nindent 8 }}
+      volumes:
+      - name: config
+        secret:
+          secretName: {{ template "keess.fullname" . }}
+          defaultMode: 420        
+      containers:
+        - name: {{ .Chart.Name }}
+          securityContext:
+            {{- toYaml .Values.securityContext | nindent 12 }}
+          image: "{{ .Values.image.repository }}:{{ .Values.image.tag | default .Chart.AppVersion }}"
+          args:
+          - run
+          - --sourceContext={{ .Values.clusterName }}
+          {{- if .Values.remoteClusters }}
+          - --destinationContexts={{ .Values.remoteClusters }}
+          {{- end }}
+          {{- if .Values.developmentMode }}
+          - --developmentMode={{ .Values.developmentMode }}
+          {{- end }}
+          imagePullPolicy: {{ .Values.image.pullPolicy }}
+          volumeMounts:
+          - name: config
+            mountPath: /root/.kube
+            readOnly: true
+          # ports:
+          #   - name: http
+          #     containerPort: {{ .Values.service.port }}
+          #     protocol: TCP
+          # livenessProbe:
+          #   httpGet:
+          #     path: /
+          #     port: http
+          # readinessProbe:
+          #   httpGet:
+          #     path: /
+          #     port: http
```

Comment:
I wouldn't like to deploy it to the cluster without this.

Reply:
Done!